Autopilot/Inc/gps.hpp

```diff
@@ -55,7 +42,56 @@ class Gps
 		* or with old data (in case of old data, the dataIsNew flag of the result struct will be cleared).
 		* @param[out]		Data 		pointer to the result struct.
 		*/
-        virtual void GetResult(GpsData_t *Data) = 0;
+        virtual void GetResult(GpsData_t &Data) = 0;
 };
 
+class NEOM8 : public Gps 
+{
+	public:
+		NEOM8(const NEOM8*) = delete;
+		NEOM8* GetInstance();
+
+		/**
+         * Triggers interrupt for new GPS measurement - stores raw data in variables and returns right away
+         * */
+		void BeginMeasuring();
+
+		 /**GetResult should:
+         * 1. Reset dataIsNew flag
+         * 2. Transfers raw data from variables to struct
+         * 3. Updates utcTime and status values in struct as well
+         * */
+		void GetResult(GpsData_t &Data);
```

Comment:
this function seems to not be implemented, there is a function called get_gps_data, but it doesent give anything back to the caler. Not sure what's going on.

Reply:
HUH!? I swear I wrote it. Must've accidentally deleted it.